frontend/src/pages/Projects/Project/ChangeRequest/FormConfig.ts

```diff
@@ -104,7 +103,6 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
       fieldType: "multiText",
       fieldLabel: "Types",
       width: "full",
-      // required: true,
```

Comment:
is this field actually no longer required?

Reply:
I'll run some tests to see if they are actually required.

it probably makes more sense to keep those comments, as @mhaswell-bcgov said to keep the comments in so we don't lose track.